test/server-specs.js

```diff
@@ -18,6 +18,14 @@ describe('server configuration', () => {
     configureServer(app, configureRoutes);
     app.use.callCount.should.equal(10);
   });
+
+  it('should reject error is thrown in configureRoutes parameter', async () => {
+    let configureRoutes = () => {
+      throw new Error("I'm Mr. MeeSeeks look at me!");
+    };
+    return server(configureRoutes, 8181).should.eventually.be
+           .rejectedWith("MeeSeeks");
```

Comment:
if we have a chaining line, let's indent it 1 beyond the indent of the thing that starts the chain